src/services/monitoring.ts

```diff
@@ -10,422 +26,527 @@ const {
-              activityReportIds: ['14AR12345'],
-              endDate: '06/24/2024',
-              topics: ['Human Resources', 'Ongoing Monitoring Management System', 'Safety Practices', 'Training and Profressional Development'],
-              status: 'Complete',
-            },
-          ],
-        },
-        {
-          citation: '1302.91(a)',
-          status: 'Active',
-          type: 'Noncompliance',
-          category: 'Program Management and Quality Improvement',
-          correctionDeadline: '09/18/2024',
-          objectives: [
-            {
-              title: 'The TTA Specialist will support the recipient by developing a plan to ensure all staff implement health and safety practices to always keep children safe',
-              activityReportIds: ['14AR12345'],
-              endDate: '06/24/2024',
-              topics: ['Human Resources', 'Ongoing Monitoring Management System', 'Safety Practices', 'Training and Profressional Development'],
-              status: 'Complete',
+    include: [
+      {
+        model: MonitoringReviewStatusLink,
+        as: 'statusLink',
+        include: [
+          {
+            model: MonitoringReviewStatus,
+            as: 'monitoringReviewStatuses',
+            required: true,
+            where: {
+              name: REVIEW_STATUS_COMPLETE,
             },
-          ],
-        },
-        {
-          citation: '1302.12(m)',
-          status: 'Active',
-          type: 'Noncompliance',
-          category: 'Program Management and Quality Improvement',
-          correctionDeadline: '09/18/2024',
-          objectives: [
-            {
-              title: 'The TTA Specialist will support the recipient by developing a plan to ensure all staff implement health and safety practices to always keep children safe',
-              activityReportIds: ['14AR12345'],
-              endDate: '06/24/2024',
-              topics: ['Human Resources', 'Ongoing Monitoring Management System', 'Safety Practices', 'Training and Profressional Development'],
-              status: 'Complete',
+          },
+        ],
+      },
+      {
+        model: MonitoringReviewLink,
+        as: 'monitoringReviewLink',
+        required: true,
+        include: [
+          {
+            model: MonitoringReviewGrantee,
+            as: 'monitoringReviewGrantees',
+            required: true,
+            where: {
+              grantNumber: grantNumbers,
             },
-          ],
-        },
-      ],
-    },
-  ];
+          },
+          {
+            model: MonitoringFindingHistory,
+            as: 'monitoringFindingHistories',
+            include: [
+              {
+                model: MonitoringFindingLink,
+                as: 'monitoringFindingLink',
+                include: [
+                  {
+                    model: MonitoringFindingStandard,
```

Comment:
Still missing the required join to MonitoringFindingGrantees, the include needs a where matching MonitoringReviewGrantees.granteeId to MonitoringFindingGrantees.granteeId

Reply:
something like (?):
```js
           {
                    model: MonitoringFindingGrant,
                    as: 'monitoringFindingGrants',
                    where: {
                      granteeId: {
                        [Op.in]: sequelize.col('monitoringReviewLink.monitoringReviewGrantees.granteeId'),
                      },
                    },
                  },

Reply:
[ae3ac83](https://github.com/HHS/Head-Start-TTADP/pull/2507/commits/ae3ac831b05cfe62d413fb6adb251e34754f5ded)